sdks/python/apache_beam/io/iobase.py

```diff
@@ -244,7 +272,7 @@ def is_bounded(self):
     return True
 
 
-class RangeTracker(object):
+class RangeTracker(Generic[PositionT]):
```

Comment:
@udim For the base `RangeTracker` if we only consider the methods implemented by the class the "position" type could be `Any`, but based on the docs it seemed that any meaningful position would be comparable/sortable.  Let me know if you think we should move the `Position` restriction down to a subclass.

Reply:
I thin Any is right here. Usually they'll be primitive, sortable types, but that's not a requirement.